includes/class-sensei-autoloader.php

```diff
@@ -285,6 +285,11 @@ public function initialize_class_file_map() {
 			\Sensei\Internal\Quiz_Submission\Submission\Repositories\Comments_Based_Submission_Repository::class => 'internal/quiz-submission/submission/repositories/class-comments-based-submission-repository.php',
 			\Sensei\Internal\Quiz_Submission\Submission\Repositories\Submission_Repository_Factory::class => 'internal/quiz-submission/submission/repositories/class-submission-repository-factory.php',
 			\Sensei\Internal\Quiz_Submission\Submission\Repositories\Submission_Repository_Interface::class => 'internal/quiz-submission/submission/repositories/class-submission-repository-interface.php',
+
+			/**
+			 * Email Customization
+			 */
+			\Sensei\Internal\Emails\Email_Post_Type::class => 'internal/emails/class-email-post-type.php',
```

Comment:
Is possible only load this one when the feature is on, and then make the email_post type auto-initializable?

Reply:
This code makes the class available for usage. It does't instantiate it.